platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/config/BenchmarkConfig.java

```diff
@@ -62,19 +59,20 @@ public record BenchmarkConfig(
         @ConfigProperty(defaultValue = "0") int csvWriteFrequency,
         @ConfigProperty(defaultValue = "false") boolean csvAppend,
         @ConfigProperty(defaultValue = "sda") String deviceName) {
+
+    @Override
     public String toString() {
-        return new ToStringBuilder(this, SHORT_PREFIX_STYLE)
-                .append("benchmarkData", benchmarkData)
-                .append("saveDataDirectory", saveDataDirectory)
-                .append("verifyResult", verifyResult)
-                .append("enableSnapshots", enableSnapshots)
-                .append("printHistogram", printHistogram)
-                .append("csvOutputFolder", csvOutputFolder)
-                .append("csvMetricsFileName", csvMetricsFileName)
-                .append("csvMetricNamesFileName", csvMetricNamesFileName)
-                .append("csvWriteFrequency", csvWriteFrequency)
-                .append("csvAppend", csvAppend)
-                .append("deviceName", deviceName)
-                .toString();
+        return "BenchmarkConfig{" + "benchmarkData='"
+                + benchmarkData + '\'' + ", saveDataDirectory="
+                + saveDataDirectory + ", verifyResult="
+                + verifyResult + ", enableSnapshots="
+                + enableSnapshots + ", printHistogram="
+                + printHistogram + ", csvOutputFolder='"
+                + csvOutputFolder + '\'' + ", csvMetricsFileName='"
```

Comment:
Are the single quotes around some of the values necessary? It appears that this is the only `toString` method which has these